### DISCUSSION TITLE
app settings debug = true log location

### DISCUSSION THREAD

**Discussion by ScottDHall** [ariacom/Seal-Report#110] [Q&A]
I'm troubleshooting an issue where the number of results returned varied from one day to another on the same data set.  In other words, day 1 it returned 400 results, and day 2 it returned 415 results.  

I'm trying to get the number of results returned for each query logged, and I set the debug = true for both appsettings json files.  Does this do anything, and if so, where would those logs be kept.

**Comment by ariacom** [comment]
Debug=true in the appsettings is to generate extra log (in Logs repository folder) to debug the Web Report Server.
You may set the 'Print Query' flag of the model concerned:
![image](https://github.com/ariacom/Seal-Report/assets/12451331/b4c4884f-5c00-421d-85f1-5ebdb2963064)
Then you can check the query sent to the database in the Log files.

**Comment by ScottDHall** [comment]
Great.  Thank you.

**Comment by ScottDHall** [comment]
Is there a way to log the returned row count?

**Reply by ariacom** [reply]
the simplest is to create an output on a directory, you will get an HTML file having the number of rows for each model in the information section...
example:
![image](https://github.com/user-attachments/assets/3cb72419-b190-43d8-88c5-f90724aa58c6)


or you may create a task that logs the information you want.
